content/en/docs/concepts/services-networking/service.md

```diff
@@ -74,8 +74,8 @@ a new instance.
 The name of a Service object must be a valid
 [DNS label name](/docs/concepts/overview/working-with-objects/names#dns-label-names).
 
-For example, suppose you have a set of Pods that each listen on TCP port 9376
-and carry a label `app=MyApp`:
+For example, suppose you have a set of Pods and each of them listens on TCP port 9376
```

Comment:
Hi @hezhizhen . Looking better.
For example, suppose you have a set of Pods where each listens on TCP port 9376 and contains a label ...

Reply:
sounds good.

Reply:
@kbhawkey Take another look?

Reply:
👀